docs/core/concepts/secrets.md

```diff
@@ -17,15 +19,32 @@ Though Prefect takes steps to ensure that `Secret` objects do not reveal sensiti
 
 ### Local Context
 
-The base `Secret` class first checks for secrets in local context, under `prefect.context.secrets`. This is useful for local testing, as secrets can be added to context by setting the environment variable `PREFECT__CONTEXT__SECRETS__FOO`, corresponding to `secrets.foo` (or `secrets.FOO`, if your OS is case-sensitive).
+The base `Secret` class first checks for secrets in local context, under `prefect.context.secrets`. This is useful for local testing, as secrets can be added to context by setting environment variables matching the configuration syntax `PREFECT__CONTEXT__SECRETS__{SECRETNAME}={SECRETVALUE}`. (Learn more about configuration in our [Configuration concept docs](configuration.md).)
 
-### Prefect Cloud
+For example, given an environment with the environment variable `PREFECT__CONTEXT__SECRETS__FOO=mypassword`, the value `"mypassword"` could be retrieved by using a `PrefectSecret` task or by using the secrets API directly, as shown below.
 
-If the secret is not found in local context and `config.cloud.use_local_secrets=False`, the base `Secret` class queries the Prefect Cloud API for a stored secret. This call can only be made successfully by authenticated Prefect Cloud Agents.
+:::: tabs
+::: tab PrefectSecret
+```python
+>>> from prefect.tasks.secrets import PrefectSecret
+>>> p = PrefectSecret('foo')                        
+>>> p.run()                   
+{'ACCESS_KEY': 'abcdef', 'SECRET_ACCESS_KEY': 'ghijklmn'}
```

Comment:
Might want to make this output the same value as the other tab

Reply:
Fixed in cc15fb5, I had too much AWS on the brain